model/database/registry.yaml

```diff
@@ -509,6 +509,11 @@ groups:
         stability: experimental
         brief: Cosmos DB sub status code.
         examples: [1000, 1002]
+      - id: db.cosmosdb.consistency_level
+        type: string
+        stability: experimental
+        brief: Account or Request Consistency Level.
+        examples: [Eventual, ConsistentPrefix, BoundedStaleness, Strong or Session]
```

Comment:
it seems it should be an enum attribute with explicitly defined members. We also probably want to have the consistent across languages and similar to other attribute values, i.e. lowercase

Reply:
I kept it as we have it in our code, I wouldn't want to change the casing while inserting this attribute (just a minor overhead)